stories/Templates/GenericPage/generic-page.scss

```diff
@@ -72,7 +88,7 @@
   .pagehero-full {
     margin-top: 0;
 
-    @include devicebreak(medium) {
+    @media (min-width:1024px){
```

Comment:
@Ashish-srij Kindly use mixin defined for media query here?

Reply:
@shishirphadke Done.